docs/faq.md

```diff
@@ -38,7 +38,7 @@ scpca_sample <- readRDS("SCPCL000000_filtered.rds")
 A sample ID, labeled as `scpca_sample_id` and indicated by the prefix `SCPCS`, represents a unique tissue that was collected from a participant.
 
 The library ID, labeled as `scpca_library_id` and indicated by the prefix `SCPCL`, represents a single set of cells from a tissue sample, or a particular combination of samples in the case of multiplexed libraries.
-For single-cell or single-nuclei experiments, this will be the result of emulsion and droplet generation using the 10X Genomics workflow, potentially including both RNA-seq, CITE-seq and cell hashing sequencing libraries.
+For single-cell or single-nuclei experiments, this will be the result of emulsion and droplet generation using the 10X Genomics workflow, potentially including both RNA-seq, ADT (i.e. from CITE-seq), and cell hashing sequencing libraries.
```

Comment:
Probably more places, but officially "10x" uses a lower case "x", so we should do that.
```suggestion
For single-cell or single-nuclei experiments, this will be the result of emulsion and droplet generation using the 10x Genomics workflow, potentially including both RNA-seq, ADT (i.e., from CITE-seq), and cell hashing sequencing libraries.
```

Reply:
I'll track them down!